setup.cfg

```diff
@@ -132,6 +132,8 @@ exclude_lines =
 
 [isort]
 # Set sorting of imports to be consistent with black formatting
-line_length=90
 multi_line_output=3
-include_trailing_comma: True
+include_trailing_comma=True
+force_grid_wrap=0
+use_parentheses=True
+line_length=88
```

Comment:
After playing around with `isort`, I'm really excited to have this implemented!!

According to `isort`, the [preferred configuration files are `.isort.cfg` or `pyproject.toml`](https://timothycrosley.github.io/isort/docs/configuration/config_files/).  Since we already have a `pyproject.toml`, I say we migrate the configuration to that file.

After playing with the `isort` options, I think this would be the closes to what we've been implementing so far...

``` toml
[tool.isort]
line_length = 88
wrap_length = 80
sections = ["FUTURE", "STDLIB", "FIRSTPARTY", "LOCALFOLDER"]
known_first_party = ["plasmapy", ]
default_section = "STDLIB"
multi_line_output = 3
force_single_line = true
use_parentheses = true
include_trailing_comma = true
force_alphabetical_sort_within_sections = true
lines_between_types = 1
```

Note, this will force third party packages and standard library packages into the same import section.  That is how we've been organizing imports thus far.  If we want to split up standard library and third party imports, then that's fine too.  I just figure we stick with what we've been doing.

Reply:
I don't really see that as something worth setting up custom settings for - I'd be happy to go for isort defaults. But since you've gone to the trouble of doing it, and you likely have Opinions about this, I'll add this config and rerun the full isort pass.